components/feed-autorized-new/FeedAuthorizedNew.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+import React from 'react';
+import { PostEditor } from '@/components/post-editor/PostEditor';
+import RecommendationsBox from '@/components/recommendations-box/RecommendationsBox';
+import {
+  ETypographyLevels,
+  Heading,
+} from '@ost-cas-fee-adv-23-24/elbmum-design';
+import PostsLoader from '@/components/posts-loader/PostsLoader';
+import { useRouter } from 'next/navigation';
+
+interface IProps {
+  userIdentifier: string;
+}
+
+const FeedAuthorizedNew = ({ userIdentifier }: IProps) => {
```

Comment:
there is a typo in the folder name of this file: 
feed-au**to**rized-new --> feed-au**tho**rized-new